ginga/canvas/types/astro.py

```diff
@@ -1276,7 +1276,7 @@ def draw(self, viewer):
             update = True
 
         cur_limits = viewer.get_limits()
-        if cur_limits != self._cur_limits:
+        if np.any(cur_limits != self._cur_limits):
```

Comment:
How about
```python
if not np.isclose(cur_limits, self._cur_limits):
``` ?

Reply:
Yes, I think for this purpose, `isclose` would work just the same?

Reply:
@pllim, I just understand `np.isclose` better than np.any, but I'm ok with either. You choose, please!

Reply:
`isclose` is more elegant! Go ahead. 😄

Reply:
I had to use `np.any` with `np.close` so it's a win-win! :smile:

Reply:
I pushed up a commit on the PR.